spread.yaml

```diff
@@ -687,7 +657,6 @@ suites:
         systems: [-ubuntu-core-*, -opensuse-*]
         # autopkgtest runs against localhost which causes problems with
```

Comment:
I think we also need to remove this comment now - autopkgtest does no longer run the "upgrade" test.

Reply:
done